cmd/ci-operator/main.go

```diff
@@ -392,8 +393,9 @@ func (o *options) Run() error {
 		log.Printf("Ran for %s", time.Now().Sub(start).Truncate(time.Second))
 	}()
 
+	dryLogger := &steps.DryLogger{}
```

Comment:
Couldn't we make this a singleton?

Reply:
I will check this. We won't have to pass the drylogger to all the steps too...Thanks.